main.js

```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
 const navPortfolio = document.getElementById('mobile__nav_portfolio');
 const navAbout = document.getElementById('mobile__nav_about');
 const navContact = document.getElementById('mobile__nav_contact');
+const cardContainer = document.getElementById('card-content');
+const cardContainer1 = document.getElementById('card-content1');
+const portFolio = document.getElementById('projects-container');
+const buttons = portFolio.getElementsByTagName('button');
+const popUp = document.getElementById('popupp');
+const generalDocument = document.getElementById('html');
+
+const projectInformation = [
+  {
+    id: 0,
+    name: '',
+    description: '',
+    feature: '',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: '',
+    sourceLink: '',
+  },
+  {
+    id: 1,
+    name: 'Data Dashboard',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'This is a very basic suuvey that can be modified at your convinience',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+  },
```

Comment:
- [ ] Kindly use the same popup window for all cards: 👇🏽 
- [ ] When I scroll down in the popup window I should see the popup's content and the back window should be fixed. I think you could use `overflow: scroll;` in the styles.

- There is a title.
- The paragraph is a "Lorem ipsum"
- The border in the orange buttons are in the same color.
- The close button should be inside the popup window
- The popup window should be responsive, mobile, and desktop versions.

| **Figma template** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/195225065-17a31764-6ecf-4c2b-97b6-34973e213edb.png) | ![image](https://user-images.githubusercontent.com/58642949/195226531-d7718558-3d8a-4934-abdf-5cdd85ae3533.png) |
| ![image](https://user-images.githubusercontent.com/58642949/195225150-f92ca2a6-7ef0-4a43-93fd-8be1ae3d094a.png) | ![image](https://user-images.githubusercontent.com/58642949/195226664-be0138ad-d289-4895-8187-1a6e83c32bd8.png) |